src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -18,58 +29,66 @@
   }
 }
 
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
+.nav-dropdown {
   position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
+  z-index: 1;
+  top: 0;
   height: auto;
-  max-height: 75vh;
-  overflow-y: auto;
-
-  padding: 20px;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.5);
-  border-radius: 0 0 24px 24px;
-  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  width: $search-results-width;
+  margin-top: 66px;
+  width: 290px;
   background-color: $saturated-black;
+  text-align: left;
+  box-shadow: 0 10px 30px 0 rgba($black, 0.5);
+  border-radius: 0 0 $border-radius--input $border-radius--input;
+  transform: translateY(-100px);
+  transform-origin: top right;
+  transition: all 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  pointer-events: none;
 
-  .nav-search-result-number {
+  &.show {
+    transform: scale(1) translateY(0px);
```

Comment:
Do we need `0px` here, or does just `0` work?

Reply:
What do we need the `scale(1)` here for?

Reply:
fixed both: https://github.com/sparkletown/sparkle/pull/1328/commits/e21e67445665543be0fab0f660743356141c30c7